rsmtool/test_utils.py

```diff
@@ -40,7 +40,8 @@ def check_run_experiment(source,
                          consistency=False,
                          skll=False,
                          file_format='csv',
-                         given_test_dir=None):
+                         given_test_dir=None,
+                         suppress_warnings_for=[]):
```

Comment:
Nice! This will be really useful for https://github.com/EducationalTestingService/rsmtool/issues/352